Dfc.CourseDirectory/Resources/AzureSearch/SearchDatasource.json

```diff
@@ -14,5 +14,22 @@
             "@odata.type":  "#Microsoft.Azure.Search.HighWaterMarkChangeDetectionPolicy",
             "highWaterMarkColumnName":  "updated"
         }
+    },
+    {
+        "name" : "sql-coursedirectory-providers",
+        "type":  "azuresql",
+        "description": "Provider list",
+        "credentials" :
+        {
+            "connectionString":  "__sqlCourseDirectoryConnectionString__"
+        },
+        "container": {
+            "name": "vwProviderSearchIndex",
+            "query": null
+        },
+        "dataChangeDetectionPolicy": {
+            "@odata.type" : "#Microsoft.Azure.Search.HighWaterMarkChangeDetectionPolicy",
+            "highWaterMarkColumnName":  "HighWaterMarkTimestamp"
```

Comment:
This doesn't seem to line up with any column in the SQL view.

Reply:
oops

Reply:
thanks, well spotted, fixed